File: J_se_les12/HW/Core/Logic/Interfaces/ILFactory.java
package Java.J_se_les12.HW.Core.Logic.Interfaces;

import Java.J_se_les12.HW.Core.Phonebook.Phonebook;

public interface ILFactory {
    void makeOperation(String path, String choose, Phonebook phonebook, IConsoleOutErrors errors, IConsoleOutput output, IConsoleOutSuc outSuc, IConsoleInput input);
}

File: J_se_les13/HW/Core/Logic/Interfaces/ILFactory.cs
using HW.Core.Phonebook;
namespace HW.Core.Logic.Interfaces
{
    public interface ILFactory
    {
        void MakeOperation(string path, string choose, PhonebookV1 phonebook, IConsoleOutError error, IConsoleOutput output, IConsoleOutSuc outSuc, IConsoleInput input);
    }
}